src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -443,12 +355,12 @@ def send_ephemeral_message(
     return {"id": response["channel"], "timestamp": response["ts"]}
 
 
-def add_pin(client: Any, conversation_id: str, timestamp: str):
+def add_pin(client: WebClient, conversation_id: str, timestamp: str) -> SlackResponse:
     """Adds a pin to a conversation."""
     return make_call(client, "pins.add", channel=conversation_id, timestamp=timestamp)
 
 
-def message_filter(message):
+def message_filter(message) -> Union[str, None]:
```

Comment:
```suggestion
def message_filter(message) -> Union[dict, None]:
```

or `dict | None`

Reply:
Woops -- it's a dict not a str. Will fix it.